build.gradle

```diff
@@ -154,6 +154,7 @@ dependencyManagement {
 }
 
 repositories {
+  mavenCentral()
```

Comment:
?

Reply:
trying it to get a green build, it means we try mavenCentral first for dependencies.
it doesn't help with plugins though which are only in bintray